commands/azureCommands/acr-build.ts

```diff
@@ -0,0 +1,115 @@
+import { ContainerRegistryManagementClient } from 'azure-arm-containerregistry/lib/containerRegistryManagementClient';
```

Comment:
I got this error while trying to queue a build, somewhere during the create a new registry steps:
Error Message: Cannot read property 'id' of undefined

Reply:
Hi Stephen,
Could you provide me with more information? Because I just queue a build and created a new ACR and I didn't get any errors.
Thanks.

Reply:
Sorry, didn't get it again, either, that's why I was vague.  Can you look for uses of ".id" in the relevant code and try to find possibilities?  If you can't find anything, I'm okay with resolving for now.

Reply:
hahahahaha!
I didn't mean it like. What I mean is that maybe for some reason the Azure subscription didn't properly load in the code. But I agree, I'll keep an eye to see if it happens to me as well.